charts/mlp/values.yaml

```diff
@@ -77,7 +77,15 @@ gitlab:
   clientId: #clientId
   clientSecret: #clientSecret
 
+# If you would like to use an external postgres database, you can connect using these credentials
+externalPostgres:
```

Comment:
nit: For the sake of consistency, can we call it `externalPostgresql`

Reply:
Ok, updated the name to externalPostgresql.